bids2openminds/mapping.py

```diff
@@ -0,0 +1,95 @@
+experimental_approach={
+  "func": "@id: https://openminds.ebrains.eu/instances/experimentalApproach/neuroimaging",
+  "dwi": [
+    "@id: https://openminds.ebrains.eu/instances/experimentalApproach/neuroimaging",
+    "@id: https://openminds.ebrains.eu/instances/experimentalApproach/neuralConnectivity",
+    "@id: https://openminds.ebrains.eu/instances/experimentalApproach/anatomy"
+  ],
+  "fmap": "@id: https://openminds.ebrains.eu/instances/experimentalApproach/neuroimaging",
+  "anat": [
+    "@id: https://openminds.ebrains.eu/instances/experimentalApproach/neuroimaging",
+    "@id: https://openminds.ebrains.eu/instances/experimentalApproach/anatomy"
+  ],
+  "perf": "@id: https://openminds.ebrains.eu/instances/experimentalApproach/neuroimaging",
+  "meg": "@id: https://openminds.ebrains.eu/instances/experimentalApproach/neuroimaging",
+  "eeg": "@id: https://openminds.ebrains.eu/instances/experimentalApproach/electrophysiology",
+  "ieeg": "@id: https://openminds.ebrains.eu/instances/experimentalApproach/electrophysiology",
+  "beh": "@id: https://openminds.ebrains.eu/instances/experimentalApproach/behavior",
+  "pet": "@id: https://openminds.ebrains.eu/instances/experimentalApproach/radiology",
+  "micr": "@id: https://openminds.ebrains.eu/instances/experimentalApproach/microscopy",
+  "nirs": "@id: https://openminds.ebrains.eu/instances/experimentalApproach/neuroimaging"
+}
+
+techniques={

```

Comment:
I assume that those who are not mapped we do miss at the moment in openMINDS? I can put those on the agenda for the next ontology meeting.